public/sw.js

```
@@ -1,206 +1,116 @@
-    // Try to get from cache first
-    const cachedResponse = await cache.match(request);
-    if (cachedResponse) {
-      return cachedResponse;
-    }
-
-    // Return offline page as last resort
-    const offlinePage = await cache.match("/offline");
-    if (offlinePage) {
+      // Otherwise, fall back to the network-first strategy.
+      return await pageStrategy.handle(args);
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)